Modules/System/SharePoint Authorization/src/UserCredentials/SharePointUserCredentials.Codeunit.al

```diff
@@ -0,0 +1,82 @@
+codeunit 9141 "SharePoint User Credentials" implements "SharePoint Authorization"
+{
+    Access = Internal;
+
+    var
+        ClientId: Text;
+        AadTenantId: Text;
+        Login: Text;
+        Password: Text;
+        AccessToken: Text;
```

Comment:
Add [NonDebuggable] to all variables, for safety.

Reply:
ok, done